gunicorn.conf.py

```diff
@@ -10,6 +11,6 @@
 secure_scheme_headers = {"X-FORWARDED-PROTO": "https"}
 timeout = 350
 worker_class = "gevent"
-workers = 4
-preload_app = True
+workers = cpu_count() + 1
```

Comment:
I'm sure there is a good reason, but I'm curious: why the +1?

Reply:
😁 I saw several resources on the "internet" suggesting 2*cpi_count()+1 as a good first start. But since we know or process is CPU intensive I scaled down a bit.